demo/docker-compose.yml

```
@@ -1,36 +1,42 @@
-version: '3.7'
+version: '3.8'
+volumes:
+  pgdata:
+    driver: local
 services:
   postgres:
-    build:
-      context: postgres
-      dockerfile: Dockerfile
-    shm_size: 256MB
+    image: postgres:15beta2
+    # https://dba.stackexchange.com/q/275378/46472
+    shm_size: 3GB
```

```suggestion
    shm_size: 256MB
```

Based on https://dba.stackexchange.com/a/275385/46472 I thought this needed to be higher than shared_buffers, but Robin set me straight.